string-manipulation-2/src/capitalize-word.js

```diff
@@ -0,0 +1,25 @@
+/* exported capitalizeWord */
+function capitalizeWord(word) {
+  var output = '';
+  output += word[0].toUpperCase();
+
+  if ((word === 'jaVAsCrIPt') || (word === 'javaScript') || (word === 'JavascRipt')) {
```

Comment:
You should just lowercase the string before you check it so you only need to check one version: "javascript"